src/services/twitter/nitter/helpers.js

```diff
@@ -65,6 +65,7 @@ function getTweetFromGraphQL(data, id) {
                   const user = entry.content?.itemContent?.tweet_results?.result?.core?.user_results?.result?.legacy;
                   user.id_str = entry.content?.itemContent?.tweet_results?.result?.core?.user_results?.result.rest_id;
                   user.id = parseInt(user.id_str, 10);
+                  user.verified = entry.content?.itemContent?.tweet_results?.result?.core?.user_results?.result?.is_blue_verified;
```

Comment:
default to false? or would be empty

Reply:
also it would be grand if we stop doing 3x traversal each time

`entry.content?.itemContent?.tweet_results?.result?.core?.user_results?.result` -> cache into variable, access it later